features/events/api/src/main/java/org/opennms/netmgt/events/api/EventConstants.java

```diff
@@ -108,6 +108,7 @@
     /**
      * The poller package association changed event UEI.
      */
+    // TODO Patrick: do we still need this?
```

Comment:
@fooker I believe we can delete this event?

Reply:
Ack. Created a separate issue for that: https://issues.opennms.org/browse/NMS-12838